src/secp256k1.js

```diff
@@ -44,10 +44,7 @@ function derivePrivateKey(seed, opts = {}) {
     return privateGen;
   }
   const publicGen = secp256k1.g.mul(privateGen);
-  // A seed can generate many keypairs as a function of the seed and a uint32.
```

Comment:
Maybe leave a comment in, so later generations looking upon these stone tablets can have an inkling of what it was like to live in our times, when we had family values.